simple_generator.py

```diff
@@ -0,0 +1,269 @@
+    accord takes 3 slots in a row, volume is decreasing
+
+    """
+    if first_step == 0:
+        return
+
+    MyMIDI.addNote(track_name, channel, first_step, order, duration, volume)
+    MyMIDI.addNote(track_name, channel, first_step, order+1, duration, volume - 20)
+    MyMIDI.addNote(track_name, channel, first_step, order+2, duration, volume - 40)
+
+    MyMIDI.addNote(track_name, channel, first_step+4, order, duration, volume)
+    MyMIDI.addNote(track_name, channel, first_step+4, order+1, duration, volume - 20)
+    MyMIDI.addNote(track_name, channel, first_step+4, order+2, duration, volume - 40)
+
+    MyMIDI.addNote(track_name, channel, first_step+7, order, duration, volume)
+    MyMIDI.addNote(track_name, channel, first_step+7, order+1, duration, volume - 20)
+    MyMIDI.addNote(track_name, channel, first_step+7, order+2, duration, volume - 40)
+
+
+# if the value is zero we write one beat of pause in melody
+def write_note(track_name, channel, value, time_n, duration, volume):
+        MyMIDI.addNote(track_name, channel, value, time_n, duration, volume)
+
+
+def play_music(music_file):
+    """
+    stream music with mixer.music module in blocking manner
+    this will stream the sound from disk while playing
+    """
+    clock = pygame.time.Clock()
+    try:
+        pygame.mixer.music.load(music_file)
+        print("Music file %s loaded!" % music_file)
+    except pygame.error:
+        print("File %s not found! (%s)" % (music_file, pygame.get_error()))
+        return
+    pygame.mixer.music.play()
+    while pygame.mixer.music.get_busy():
+        # check if playback has finished
+        clock.tick(30)
+
+
+def ful_music_play(music_file):
+    print("playing music")
+    freq = 40100  # audio CD quality
+    # So, here was 44100 but it gain very strange messages, but music was in better quality
+    # ALSA lib pcm.c:7963:(snd_pcm_recover) under run occurred
+    bitsize = -16  # unsigned 16 bit
+    channels = 1  # 1 is mono, 2 is stereo
+    buffer = 1024  # number of samples
+    pygame.mixer.init(freq, bitsize, channels, buffer)
+    # optional volume 0 to 1.0
+    pygame.mixer.music.set_volume(0.8)
+    try:
+        play_music(music_file)
+    except KeyboardInterrupt:
+        # if user hits Ctrl/C then exit
+        # (works only in console mode)
+        pygame.mixer.music.fadeout(1000)
+        pygame.mixer.music.stop()
+        raise SystemExit
+
+
+def write_full(fl_melody):
+    """
+    this function takes all melody flows and writes them into file one after another
+    if first number is 0 -> it is common melody, 1 -> accord-melody
+    :param fl_melody: list of melodies
+    :return: none
+    """
+    for seq in fl_melody:
+        if seq[0] == 0:
+            write_sequence(seq)
+        if seq[0] == 1:
+            write_sequence_ac(seq)
+
+
+melody_length = 0
+while True:
+    melody_length_input = input("number of notes:")
+    if str.isnumeric(melody_length_input):
```

Comment:
AttributeError: type object 'str' has no attribute 'isnumeric'

Reply:
Hm, works for me